### PR TITLE
[forge] Tunings for realistic_network_tuned_for_throughput

### PR DESCRIPTION
**Pull request by msmouse**
### Description

Tunings used for getting some big TPS numbers

### Test Plan

https://github.com/aptos-labs/aptos-core/actions/runs/6489091369

### CODE REVIEW

**Review by grao1991 — APPROVED** [review]

File: testsuite/forge-cli/src/main.rs [hunk 1]
```diff
@@ -1693,53 +1697,54 @@ fn realistic_env_max_load_test(
 }
 
 fn realistic_network_tuned_for_throughput_test() -> ForgeConfig {
```
Comment (sitalkedia) [inline testsuite/forge-cli/src/main.rs:1699]: How about adding a new test and leave the current one unchanged?

Comment (msmouse) [reply testsuite/forge-cli/src/main.rs:1699]: The original runs at 6k iirc, I don't think it's useful or establish any baseline?

Comment (msmouse) [reply testsuite/forge-cli/src/main.rs:1699]: Or I'm missing how it's used currently? You and @igor-aptos  track it anywhere?

Comment (sitalkedia) [reply testsuite/forge-cli/src/main.rs:1699]: I thought the original runs at ~8k TPS and is included in the list of stable forge test that runs daily.. @igor-aptos can confirm..

Comment (msmouse) [reply testsuite/forge-cli/src/main.rs:1699]: Good catch then. Gonna update.

Comment (msmouse) [reply testsuite/forge-cli/src/main.rs:1699]: Or maybe just comment out the resource ask out and update the numbers in CI? ( I have difficulty naming things. :trollface: 
 )

**Review by sitalkedia — APPROVED** [review]

**Pull request merged**